apps/server/src/modules/server/admin-api.server.module.ts

```diff
@@ -0,0 +1,71 @@
+import { MikroOrmModule } from '@mikro-orm/nestjs';
+import { DynamicModule, MiddlewareConsumer, Module, NestModule } from '@nestjs/common';
+import { ALL_ENTITIES } from '@shared/domain';
+import { DB_PASSWORD, DB_URL, DB_USERNAME, createConfigModuleOptions } from '@src/config';
+import { LegacyLogger, LoggerModule } from '@src/core/logger';
+import { ConfigModule } from '@nestjs/config';
+import { RabbitMQWrapperModule, RabbitMQWrapperTestModule } from '@src/infra/rabbitmq';
+import { MongoDatabaseModuleOptions, MongoMemoryDatabaseModule } from '@src/infra/database';
+import { DeletionModule } from '../deletion';
+import { defaultMikroOrmOptions, setupSessions } from './server.module';
+import { serverConfig } from './server.config';
+import { AdminApiServerController } from './controller/admin-api-server.controller';
```

Comment:
Please export the `admin-api-server.controller.ts` via the `controller`'s dir barrel file (`index.ts`) and adjust the import here.